.travis.yml

```diff
@@ -84,6 +84,9 @@ jobs:
     script:
       - tox -e system_tests
     deploy:
+      - before_install:
+          rvm install 3.1
+          rvm use 3.1 --default
```

Comment:
Could you add a comment to the file itself to give a hint why we need this? I think our future selfs might be grateful

Reply:
Sure,I have updated the file with the comment